client-side.md

```diff
@@ -29,23 +26,9 @@ free to justify why not.
 
 ## Dependencies
 
-- Make all your dependencies explicit using a dependency system
-  (e.g. AMD, CJS, ES6 modules, @import), rather than expecting them to
-  be provided by the environment.
 - If starting a new project, favour ES6 modules unless you can justify
   otherwise.
-- Do not commit your dependencies into Git. Instead, lock your dependencies (direct and transitive) using a lockfile or shrinkwrap file, e.g. jspm’s `config.js` or npm’s `shrinkwrap.json`.
-
-*TODO: How to publish your own libraries? What format to publish it in?
-NPM, Bower, JSPM, etc? How to depend on other libraries?*
-
-Favour the following libraries unless you can justify otherwise:
-- [lodash](https://lodash.com/) over [underscore](http://underscorejs.org/)
-- [reqwest](https://github.com/ded/reqwest) for AJAX
-- [moment](http://momentjs.com/) for dates
-
-*TODO: any others?*
-
+- Do not commit your dependencies into Git. Instead, lock your dependencies (direct and transitive) using a lockfile.
```

Comment:
In some circumstances committing your dependencies can be helpful (e.g. committing the Yarn binary using [`yarn policies`](https://classic.yarnpkg.com/en/docs/cli/policies))

Reply:
Is that useful as a generalised recommendation or is it fairly specific to certain use-cases?

Reply:
Yeah, it's very specific. Maybe we could soften to:

```suggestion
- Prefer locking your dependencies (direct and transitive) using a lockfile rather than committing them into Git.
```